.gitignore

```diff
@@ -87,3 +84,11 @@ examples/contrib/obj
 examples/dotnet/bin
 examples/dotnet/obj
 
+# cmake code gen (assumes -Bbuild)
+.cmake/
+CMakeCache.txt
+CMakeFiles
+CMakeLists.txt.user
```

Comment:
technically `CMakeLists.txt.user` is generated by qtcreator aka an IDE and not by cmake (e.g. calling cmake configure/build from using a terminal)

Reply:
OK, thanks for straightening me out. Moving that file was an over-reach based purely on its name!